doc/src/manual/noteworthy-differences.md

```diff
@@ -197,14 +197,19 @@ For users coming to Julia from R, these are some noteworthy differences:
 
 ## Noteworthy differences from Python
 
-  * Julia requires `end` to end a block. Unlike Python, Julia has no `pass` keyword.
+  * Julia's `for`, `if`, `while`, etc. blocks are terminated by the `end` keyword. Indentation level
+    is not significant as it is in Python. Unlike Python, Julia has no `pass` keyword.
+  * Strings are denoted by double quotation marks (`"text"`) in Julia (with three double quotation marks for multi-line strings), whereas in Python they can be denoted either by single (`'text'`) or double quotation marks (`"text"`). Single quotation marks are used for characters in Julia (`'c'`).
+  * String concatenation is done with `*` in Julia, not `+` like in Python. Analogously, string repetition is done with `^`, not `*`.
+  * Python Lists correspond to Julia Arrays of type Any (flexible, but slow). Numpy Arrays correspond to typed Julia Arrays.
```

Comment:
```suggestion
  * Python Lists correspond to Julia `Vector{Any}` (flexible, but slow). Numpy arrays correspond to typed Julia Arrays.
```

Also, "Numpy arrays correspond to typed Julia Arrays" is kind of not technically true as `dtype` can be `object`?

Reply:
Suggestion: 

> Numpy arrays (with concrete data types, not `object`) correspond...

Reply:
I'm not sure if Python programmers are familiar with the notion of concrete type.  Also, even in Julia, array of concrete non-isbits objects can be as slow as `Array{Any}`.

So maybe something like this?

```suggestion
  * Python Lists correspond to Julia `Vector{Any}` (flexible, but slow). Machine-native arrays such as Numpy arrays can be represented by built-in type `Array` in Julia.
```

Not sure if "Machine-native arrays" is the right word though. I just wanted to say `isbits` element type without saying it. Maybe just "Fast arrays?"

Reply:
What about this formulation?

> Fast arrays such as Numpy arrays with machine-native data types (like `np.int64`, `np.float64`) can be represented by the built-in type `Array` with fixed `isbits` element type in Julia.

Reply:
I think the complication is that both Numpy and Julia can store "struct types" but machine-native may sound like "scalar" types like `Float64` (i.e., the data types with corresponding machine register).

Maybe something like this?

> "Fast" arrays such as Numpy arrays that store elements in-place can be represented by ...

Maybe adding examples can help

> ... that store elements in-place (i.e., `dtype` is `np.float64`, `[('f1', np.uint64), ('f2', np.int32)]`, etc.) can be represented by built-in type `Array` in Julia (i.e., `eltype` is a `isbitstype` like `Float64`, `NamedTuple{(:f1, :f2), Tuple{UInt64, Int32}}`, etc.).

Reply:
This point got more complicated than I initially thought, I was not aware that it is possible to store structs in-place with numpy.
Adding the examples would certainly help imho.

Reply:
updated using your suggestion